libsolidity/interface/FileReader.cpp

```diff
@@ -73,7 +73,7 @@ ReadCallback::Result FileReader::readFile(string const& _kind, string const& _so
 			return ReadCallback::Result{false, "File outside of allowed directories."};
 
 		if (!boost::filesystem::exists(canonicalPath))
-			return ReadCallback::Result{false, "File not found."};
+			return ReadCallback::Result{false, "File not found: " + canonicalPath.generic_string()};
```

Comment:
Why generic?

Reply:
I've changed that to `string()`. (I chose generic_string over string because that's what we used above. but I think here string is better. yes.)